src/components/ReplyComment/index.js

```diff
@@ -95,6 +95,7 @@ const ReplyCommentId = ({
       <ScrollView ref={scrollViewRef} contentContainerStyle={styles.commentScrollView}>
         <View style={styles.containerComment}>
           <ReplyCommentItem
+            feedId={dataFeed.id}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `feedId` prop is being passed to the `ReplyCommentItem` component. Ensure that this prop is properly used and validated within the `ReplyCommentItem` component. If it's not optional, you should also handle scenarios where it might be `undefined` or `null`.
```diff
+ feedId={dataFeed.id}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->